.github/workflows/delete-pool-on-pr-close.yaml

```diff
@@ -0,0 +1,63 @@
+name: Tear down Batch pool
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
Do we need `workflow_dispatch`? (That would give us the option to run this manually, which I'm not sure we need?)

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/125/commits/7b89487b91a9fcaf027f68902cfbe91c343edfdf